osx/2_build_toolchain.sh

```diff
@@ -44,17 +44,22 @@ function set_build_flags {
 	CLANGXX=`xcodebuild -find clang++`
 	ARCH="-arch x86_64"
 	SDKPATH=`xcrun -sdk macosx --show-sdk-path`
+	SYSROOT="-mmacosx-version-min=10.9 -isysroot $SDKPATH"
 
 	export CC="$CLANG $ARCH"
 	export CXX="$CLANGXX $ARCH"
 	if [ "$ENABLE_CCACHE" ]; then
 		export CC="ccache $CC"
 		export CXX="ccache $CXX"
 	fi
-	export CFLAGS="-g -O2 -mmacosx-version-min=10.9 -isysroot $SDKPATH"
-	export CXXFLAGS=$CFLAGS
-	export CPPFLAGS="-I$PLATFORM_PREFIX/include"
-	export LDFLAGS="-L$PLATFORM_PREFIX/lib $ARCH -mmacosx-version-min=10.9 -isysroot $SDKPATH"
+	export CFLAGS="-g -O2 $SYSROOT"
+	export CPPFLAGS="-I$PLATFORM_PREFIX/include $SYSROOT"
+	export LDFLAGS="-L$PLATFORM_PREFIX/lib $ARCH $SYSROOT"
+
+	# For freetype. apinames ignores CC and CFLAGS because it requires
+	# the host compiler but macos Catalina lacks /usr/include
+	export CCexe="$CC"
+	export CCexe_CFLAGS="$CPPFLAGS"
```

Comment:
Freetype with there custom build system trying to be smart... Checking if we are cross-compiling would be too simple.